elements/AAC-850m6/update.js

```diff
@@ -426,6 +436,22 @@ var update = function(instance, properties, context) {
 
       //add Quill container div to page
       instance.canvas.append(`<div id="${id}"></div>`);
+
+      // Hack the clipboard to bypass the annoying re-focus
+      // which centers the top of the Quill container after paste
+      const Clipboard = Quill.import('modules/clipboard')
```

Comment:
missing semicolons here?

Reply:
yeah it's a bit inconsistent but our standard JS doesn't use semicolons, so I took the habit of removing them wherever I went. now this file may have some and be missing some but you're right that it mostly has them so I'll add them back for consistency